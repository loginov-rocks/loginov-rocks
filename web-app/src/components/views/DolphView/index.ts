import { DolphView } from './DolphView';

export { DolphView };
