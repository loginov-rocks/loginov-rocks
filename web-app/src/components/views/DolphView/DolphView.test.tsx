/* eslint-disable import/no-extraneous-dependencies */

import { shallow } from 'enzyme';
import * as React from 'react';

import { DolphView } from './DolphView';

it('matches snapshot', () => {
  const wrapper = shallow(<DolphView />);

  expect(wrapper).toMatchSnapshot();
});
