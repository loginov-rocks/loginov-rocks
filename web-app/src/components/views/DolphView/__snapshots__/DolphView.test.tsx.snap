// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Fragment>
  <h1>
    Hello!
  </h1>
  <p>
    I am
     
    <strong>
      Dolph
    </strong>
    , I do not bite, but worry a lot and miss my family!
  </p>
  <p>
    Please call them or write on Telegram or WhatsApp:
     
    <PhoneLink
      first="7 (9"
      fourth="9-08"
      second="95) 9"
      third="97-0"
      title="(Alla)"
    />
     
    or
     
    <PhoneLink
      first="7 (9"
      fourth="0-19"
      second="95) 9"
      third="97-1"
      title="(Danila)"
    />
    !
  </p>
  <p>
    On
     
    <strong>
      March 24, 2022
    </strong>
    , I am traveling by plane with my mom
     
    <strong>
      Alla Loginova
    </strong>
     
    from Saint Petersburg (Russia) through Istanbul (Turkey) to Tbilisi (Georgia):
  </p>
  <ul>
    <li>
      Turkish Airlines
       
      <a
        href="https://www.flightradar24.com/data/flights/tk402"
        rel="noreferrer"
        target="_blank"
      >
        TK 402
      </a>
      : LED (Saint Petersburg) 15:00 — IST (Istanbul) 18:55
    </li>
    <li>
      Turkish Airlines
       
      <a
        href="https://www.flightradar24.com/data/flights/tk386"
        rel="noreferrer"
        target="_blank"
      >
        TK 386
      </a>
      : IST (Istanbul) 21:25 — TBS (Tbilisi) 00:35
    </li>
  </ul>
  <p>
    Microchip #
     
    <strong>
      643 094 100 473 529
    </strong>
  </p>
</Fragment>
`;
